autobotAI_integrations/integrations/azure_devops/__init__.py

```
@@ -0,0 +1,305 @@
+
+    def __init__(self, **kwargs):
+        # If organization is provided, use it as accountId fallback if accountId is not explicitly set.
+        if kwargs.get("organization"):
+            kwargs.setdefault("accountId", kwargs["organization"])
+        kwargs.setdefault("cspName", "AZURE_DEVOPS")
+        kwargs.setdefault("alias", kwargs.get("organization") or "Azure DevOps")
+        super().__init__(**kwargs)
+
+    @classmethod
+    @field_validator("base_url")
+    def validate_base_url(cls, v):
```

Ruff (0.15.21)

[warning] Missing return type annotation for classmethod `validate_base_url`

(ANN206)